packages/mobile/src/redux/store.ts

```diff
@@ -1,21 +1,51 @@
 import AsyncStorage from '@react-native-community/async-storage'
 import { applyMiddleware, compose, createStore } from 'redux'
-import { createMigrate, persistReducer, persistStore } from 'redux-persist'
+import { createMigrate, getStoredState, persistReducer, persistStore } from 'redux-persist'
+import FSStorage from 'redux-persist-fs-storage'
 import autoMergeLevel2 from 'redux-persist/lib/stateReconciler/autoMergeLevel2'
 import createSagaMiddleware from 'redux-saga'
+import { PerformanceEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { APP_NAME } from 'src/config'
 import { migrations } from 'src/redux/migrations'
 import rootReducer from 'src/redux/reducers'
 import { rootSaga } from 'src/redux/sagas'
+import { ONE_DAY_IN_MILLIS } from 'src/utils/time'
+
+const timeBetweenStoreSizeEvents = ONE_DAY_IN_MILLIS
+let lastEventTime = Date.now()
 
 const persistConfig: any = {
   key: 'root',
   version: 7, // default is -1, increment as we make migrations
-  storage: AsyncStorage,
+  keyPrefix: `${APP_NAME}-`, // the redux-persist default is `persist:` which doesn't work with some file systems.
```

Comment:
Let's use something that really can't change, if we ever change `APP_NAME` this would break existing clients. 😅